include/COSMO_SAC/COSMO.hpp

```diff
@@ -518,7 +518,8 @@ namespace COSMOSAC {
                 w = -0.27027;
             }
 
-            double ekB0 = m_fluids[0].dispersion_eoverkB, ekB1 = m_fluids[1].dispersion_eoverkB;
+            double ekB0 = m_fluids[0].dispersion_eoverkB.value_or(0.0);
+            double ekB1 = m_fluids[1].dispersion_eoverkB.value_or(0.0);
```

Comment:
Replaced direct access to `dispersion_eoverkB` with `value_or(0.0)` to handle optional values and ensure a default value of 0.0 is used if `dispersion_eoverkB` is not set. This prevents potential null value issues.